www/templates/html/MediaPlayer.tpl.php

```diff
@@ -68,9 +68,14 @@
 
                         t.captionsButton.before($myButton)
 
-                        t.controls.on("click", ".caption-toggle", function(){
+                        t.controls.on("click", ".caption-toggle", function(e){
                             $captionSearch.toggleClass("show");
                         });
+                        t.controls.on("keyup", ".caption-toggle", function(e){
+                            if(e.keyCode != 13){
+                                $captionSearch.addClass("show");
```

Comment:
This makes it so that the searchable caption box pops open if you are tabbing through the video player. Except on return because that also fires the click event and it opens and closes very quickly.